docs/releasehistory.rst

```diff
@@ -7,6 +7,78 @@ Releases follow the ``major.minor.micro`` scheme recommended by `PEP440 <https:/
 * ``minor`` increments add features but do not break API compatibility
 * ``micro`` increments represent bugfix releases or improvements in documentation
 
+
+
+0.4.0 - Performance optimizations and support for SMIRNOFF 0.3 specification
+----------------------------------------------------------------------------
+
+This update improves runtime when running ``create_openmm_system`` on large topologies.
+
+This update also introduces the `SMIRNOFF 0.3 specification <https://open-forcefield-toolkit.readthedocs.io/en/0.4.0/smirnoff.html>`_.
+The spec update is the result of discussions about how to handle the evolution of data and parameter types as further functional forms are added to the SMIRNOFF spec.
+
+
+We provide methods to convert SMIRNOFF 0.1 and 0.2 forcefields written with the XML serialization (.offxml) to the SMIRNOFF 0.3 XML specification.
+These methods are called automatically when loading a serialized SMIRNOFF data representation written in the 0.1 or 0.2 specification.
+This functionality allows the toolkit to continue to read files containing SMIRNOFF 0.2 spec force fields, and also implements backwards-compatibility for SMIRNOFF 0.1 spec force fields.
+
+
+WARNING: The SMIRNOFF 0.1 spec did not contain fields for several energy-determining parameters that are exposed in later SMIRNOFF specs.
```

Comment:
This should be a `.. warning ::` block.

Reply:
Changed for this release. 

Generally, I'm nervous about putting in too much ReStructuredText-specific formatting here if it's not absolutely necessary. Since we will be copying these to the GitHub release text (which uses MarkDown), each element that needs reformatting adds a little avoidable complexity.

Are there RST-specific features in this document that we couldn't afford to lose, or could we convert the releasenotes to markdown?